ras/lshw.py

```diff
@@ -178,8 +178,15 @@ def test_prod_id_serial(self):
         else:
             product_path = '/proc/device-tree/model'
             serial_path = '/proc/device-tree/system-id'
-        product_name = genio.read_file(product_path).rstrip(' \t\r\n\0')
-        serial_num = genio.read_file(serial_path).rstrip(' \t\r\n\0')
+
+        if 'x86_64' in process.system_output('lscpu', shell='true').decode():
```

Comment:
please use avocado native library for same like using https://github.com/avocado-framework/avocado/blob/master/avocado/utils/cpu.py#L165 

getarch() funtion ,

Reply:
even check if dmidecode command manipulation can also be push in utils , let me check , this can be pursue as enhancement

Reply:
Used get_arch from utils.